english/BayesTheorem/priors/chocolateOrSleep.md

```diff
@@ -0,0 +1 @@
+Would you like some more [chocolate](../../search_for_chocolate/search_for_chocolate.md) or would you like to go to [sing](../../sing/sing.md)?
```

Comment:
This line contains more than one link.  Please put links on separate lines with a blank line in between so that options will be visually separated.